prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="/prep-html-links/index.html">
+    <div>Click here to go to the home page</div>
```

Comment:
Oh, much better. Last thing, you can change this `href="/prep-html-links/index.html"` to `href="../index.html"`

That way your web pages do not rely on their directory being named `prep-html-links`